pkg/storagev2/migration/migration.go

```diff
@@ -0,0 +1,112 @@
+package migration
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sort"
+	"unsafe"
+
+	storage "github.com/ethersphere/bee/pkg/storagev2"
+)
+
+type Version = uint64
+type StepFn func(storage.Store) error
+type StepsMap = map[Version]StepFn
```

Comment:
we usually group these in one
```suggestion
type (
    Version = uint64
    StepFn func(storage.Store) error
    StepsMap = map[Version]StepFn
)
```
in regards to the type aliases, perhaps we should avoid them without concrete necessity: https://yourbasic.org/golang/type-alias/